pypeit/specobj.py

```diff
@@ -189,6 +189,8 @@ class SpecObj(datamodel.DataContainer):
                                    descr='Object ID for echelle data. Each object is given an '
                                          'index in the order it appears increasing from from left '
                                          'to right. These are one based.'),
+                 # TODO ECH_ORDERINDX should be purged. It is not reliable for anything given masking. Instead
+                 # one needs to use SLITID or ECH_ORDER
```

Comment:
I checked and it still appears in a few places. My inclination is to leave this in until we are further along with Echelle for HIRES ADAP at which point it will have to be purged to deal with masked orders.